request_a_govuk_domain/request/forms.py

```diff
@@ -298,20 +298,21 @@ class RegistryDetailsForm(forms.Form):
         label="Role name",
     )
 
-    registrant_contact_email = forms.CharField(
-        label="Email address",
-        help_text="Use a role-based email address, like itsupport@[yourorganisation].gov.uk",
-        validators=[
-            EmailValidator(
-                "Enter the registrant's role-based email address in the correct format, like itsupport@organisation.gov.uk"
-            )
-        ],
-    )
-
     def __init__(self, *args, **kwargs):
+        self.hint_email = kwargs.pop("hint_email", None)
         super().__init__(*args, **kwargs)
         self.helper = FormHelper()
         self.helper.label_size = Size.SMALL
+        self.fields["registrant_contact_email"] = forms.CharField(
+            label="Email address",
+            help_text=f"Use a role-based email address, like {self.hint_email}.gov.uk",
```

Comment:
The hint returns e.g. clerk@[yourorganisation].gov.uk so will this not create a string that is "Use a role-based email address, like clerk@[yourorganisation].gov.uk.gov.uk", i.e. duplicate .gov.uk? maybe I misunderstand how the hints work?